drivers/sensor/ms56xx/Kconfig

```diff
@@ -0,0 +1,60 @@
+# MS56XX pressure sensor configuration options
+
+# Copyright (c) 2019 Thomas Schmid <tom@lfence.de>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MS56XX
+	bool "MS56XX pressure and temperature sensor"
```

Comment:
not sure if you want to be more consistent: sometimes the driver is referred as being for "MS5607 and MS5611 ... " (ex.  in dts bindings), and sometimes as "MS56xx". Don't have a strong preference but it would be good to be consistent.